src/test/java/com/vmware/vip/i18n/TranslationMessageTest.java

```diff
@@ -261,6 +256,23 @@ public void testGetAllComponentTranslation() {
         Assert.assertTrue(list.size() > 0);
     }
 
+    @Test
+    public void testGetAllComponentTranslationMixedMode() {
+        String offlineResourcesBaseUrlOrig = vipCfg.getOfflineResourcesBaseUrl();
+        vipCfg.setOfflineResourcesBaseUrl("offlineBundles/");
+        List<DataSourceEnum> msgOriginsQueueOrig = vipCfg.getMsgOriginsQueue();
+        vipCfg.setMsgOriginsQueue(new LinkedList<DataSourceEnum>(Arrays.asList(DataSourceEnum.VIP, DataSourceEnum.Bundle)));
+        String vipServerOrig = vipCfg.getVipServer();
+        vipCfg.setVipServer("http://1.1.1.1:80");
+
+        List<Map> list = new ProductService(dto).getAllComponentTranslation();
+        Assert.assertTrue(list.size() > 0);
+
+        vipCfg.setOfflineResourcesBaseUrl(offlineResourcesBaseUrlOrig);
+        vipCfg.setMsgOriginsQueue(msgOriginsQueueOrig);
+        vipCfg.setVipServer(vipServerOrig);
```

Comment:
What's the reason for setting these properties again?

Reply:
@huihuiw01 , for the next tests. Tests share the same configurations, unless modified inside the test.